src/main/kotlin/com/modules/db/suspendTransaction.kt

```diff
@@ -0,0 +1,27 @@
+package com.modules.db
+
+import kotlinx.coroutines.Dispatchers
+import org.jetbrains.exposed.sql.Transaction
+import org.jetbrains.exposed.sql.transactions.experimental.newSuspendedTransaction
+/**
+ * Suspends the current coroutine until the transaction is completed.
+ * Function is generic, meaning it can return any type T.
+ *
+ * --> **block** parameter is a lambda with a receiver of
+ * type Transaction that returns a value of type T
+ * ====== Transaction.(): This part specifies that the lambda has a receiver of type Transaction.
+ * ====== Inside the lambda, you can call methods and access properties of Transaction directly.
+ * ====== the receiver is an instance of the Transaction class
+ *
+ * --> **newSuspendTransaction** starts a new database transaction
+ * in a coroutine context, it executes the block lambda
+ *
+ * --> **Dispatchers.IO**  is a coroutine dispatcher in Kotlin that is optimized for
+ * offloading blocking IO tasks to a shared pool of threads.
+ * It is designed to handle IO-bound operations such as file I/O, network requests
```

Comment:
w publicznej dokumentacji (a tym są komentarze typu `/**`) nie powinno się opisywać takich szczegółów implementacyjnych. Poza ty mnie wiem, do kogo to ma być skierowane, bo jeśli ktoś nie wie, czym jest Dispatchers.IO, to może się tego dowiedzieć z dokumentacji tegoż...